README.md

```diff
@@ -1,4 +1,4 @@
-# Go driver for [Babelfish](https://github.com/bblfsh/bblfshd) ![Driver Status](https://img.shields.io/badge/status-beta-dbd25c.svg) [![Build Status](https://travis-ci.org/bblfsh/go-driver.svg?branch=master)](https://travis-ci.org/bblfsh/go-driver) ![Native Version](https://img.shields.io/badge/go%20version-1.10-aa93ea.svg) ![Go Version](https://img.shields.io/badge/go%20version-1.12-63afbf.svg)
+# Go driver for [Babelfish](https://github.com/bblfsh/bblfshd) ![Driver Status](https://img.shields.io/badge/status-beta-dbd25c.svg) [![Build Status](https://travis-ci.org/bblfsh/go-driver.svg?branch=master)](https://travis-ci.org/bblfsh/go-driver) ![Native Version](https://img.shields.io/badge/go%20version-1.12-aa93ea.svg) ![Go Version](https://img.shields.io/badge/go%20version-1.12-63afbf.svg)
```

Comment:
Did the digest really not change on these links? They seem to work, but I don't really know how to tell what those hashes are supposed to be.

Reply:
I also though those are digests initially, but those seem to be color codes :)